hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/retryReasonCategories/ServerErrorRetryReason.java

```diff
@@ -56,10 +58,14 @@ String getAbbreviation(final Integer statusCode,
           splitedServerErrorMessage)) {
         return EGRESS_LIMIT_BREACH_ABBREVIATION;
       }
-      if (OPERATION_BREACH_MESSAGE.equalsIgnoreCase(
+      if (TPS_OVER_ACCOUNT_LIMIT.getErrorMessage().equalsIgnoreCase(
           splitedServerErrorMessage)) {
         return OPERATION_LIMIT_BREACH_ABBREVIATION;
```

Comment:
Could you please rename `OPERATION_LIMIT_BREACH_ABBREVIATION` to `TPS_LIMIT_BREACH_ABBREVIATION`, this would maintain consistency with the naming convention with other constants.

Reply:
Taken